README.md

```diff
@@ -6,7 +6,11 @@ A Discord test bot used to test out the architecture and implementation of a Dis
 
 ## Contributing
 
-So, you want to contribute to this repo? We are glad to recieve PRs from anybody who is interested. Follow these steps if you are new to contributing to open source projects:
+So, you want to contribute to this repo? We are glad to recieve PRs from anybody who is interested.
+
+The easiest way to contribute to the development of the bot is using [Gitpod](http://gitpod.io/#https://github.com/bradtaniguchi/discord-bot-test). It starts up an already working version of the bot with all the code so you can start working right now! (You'll have to add the `TOKEN` variable in the `.env` file)
+
+Follow these steps if you are new to contributing to open source projects:
```

Comment:
I'm not sure if we should say "you can start working right now!" as the user still needs to setup the discord bot in discord. Maybe we should primarily mention how gitpod can be used as a development environment instead of running things locally. Contributors will still need to setup their own discord bot on their own end though to help contribute (and deal with all the annoying aspects of that side of things!)

Reply:
Hmmm... this seems reasonable. I think I'll throw in the Gitpod logo too.

Reply:
Should we wait for the gitpod logo, or should we be good to go without it? :)

Reply:
Ooops! Completely forgot about this PR.... my apologies. We can add a logo quite easily... but do you think the README instructions are good?

Reply:
The current words should be good

Not a big deal BTW :D